### DISCUSSION TITLE
Getting stuck in some weird input mode

### DISCUSSION THREAD

**Discussion by bodograumann** [qutebrowser/qutebrowser#6403] [Q&A] [answered]
Not sure if this is a bug or intentional and where it is comming from. Maybe somebody knows more about it.

Lately it happens to me some times, that I get stuck in some weird input mode.
I can still leave the usual qutebrowser input mode with Esc, but only enter it by clicking in an input field
and most qutebrowser keybindings don't work anymore.
I can go to the command line with `:`, or close a tab with `Ctrl-w`, but not with `d`.
Also, when I enter something into a text box, the word that I currently type is underlined.
When I enter the apostrophy character, the cursor stays on the left side of it.
This is really confusing.

The only way to fix it afaics is restarting qutebrowser.
I haven’t seen it in other applications.
Still on qutebrowser 2.1.1. Changelog didn’t mention anything related though.

Does anybody know where this is coming from?

**Comment by The-Compiler** [comment, marked as answer]
I've never seen that happen. From "Also, when I enter something into a text box, the word that I currently type is underlined." this sounds more like an IME you're using is doing something weird. What is `QT_IM_MODULE` set to in your environment?

**Reply by bodograumann** [reply]
Thanks that pointed me into the right direction.
I have uim configured and that is using Shift+Space as toggle keys.
Not sure why I even have it :thinking: 
Is it needed for XCompose to work?

I remember being on xim before, but having problems (with gncash?) making me switch...

**Reply by The-Compiler** [reply]
X compose keys definitely work without any special IME.

**Reply by bodograumann** [reply]
It might have been necessary to have long compose sequences with GTK2 twelve years ago :rofl: 
Guess I can disable it now.
